Huion_WH1409/index.md

```diff
@@ -0,0 +1,20 @@
+---
+VID: "256c"
+PID: "006e"
+vendor: Huion
+product: WH1409
+image: Huion_WH1409
+report_rate: 230
+working_area:
+    width: 13.8
+    height: 8.6
+    resolution: 5080
+pen:
+    pressure_levels: 2048
+frame_controls: 12 Press Keys
+sold_as:
+    - Huion WH1409
+    - Huion Inspiroy WH1409 Graphics Drawing Tablet and 8GB MicroSD Card
```

Comment:
I don't think " and 8GB MicroSD Card" should be a part of the tablet name, could you please remove that?